NitroxServer/ConsoleCommands/Processor/ConsoleCommandProcessor.cs

```diff
@@ -51,20 +55,27 @@ public void ProcessCommand(string msg)
                 return;
             }
 
-            RunCommand(cmd, parts);
+            RunCommand(cmd, parts, player);
         }
 
-        private void RunCommand(Command command, string[] parts)
+        private void RunCommand(Command command, string[] parts, Player player)
         {
-            string[] args = parts.Skip(1).ToArray();
-
-            if (command.VerifyArgs(args))
+            //Verify is an admin or attempting to login
+            if (player.isAdmin || (command.Name == "login"))
             {
-                command.RunCommand(args);
-                return;
-            }
+                string[] args = parts.Skip(1).ToArray();
 
-            Log.Info("Command Invalid: {0}", command.Args);
+                if (command.VerifyArgs(args))
+                {
+                    command.RunCommand(args, player);
+                    return;
+                } else
+                {
+                    Log.Info("Command Invalid: {0}", command.Args);
+                    return;
+                }
+            } 
+            Log.Info("Someone without the required permissions attempted a command");
```

Comment:
You could add player information so server owner knows who tried to do a command.